Sources/RoktUXHelper/UI/Components/GroupedDistributionComponent.swift

```diff
@@ -355,4 +362,14 @@ struct GroupedDistributionComponent: View {
             }
         }
     }
+    
+    func setRecalculatedCurrentGroup() {
+        if currentLeadingOffer + viewableItems > totalOffers - 1 {
```

Comment:
for my understanding when would this happen?

Reply:
Good catch actually 😅 I retested this and this isn't applicable for grouped distribution because it is perfectly fine for the number of children in a group can be less than viewableItems. Updated logic